.github/workflows/fast-forward-candidate-branch.yml

```diff
@@ -0,0 +1,21 @@
+name: Fast-forward candidate branch to HEAD of default branch
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'The release version number e.g (2.11 or 2.12)'
+        required: true
+      ref:
+        description: 'The SHA1 or branch name the candidate branch will point to'
+        required: true
+        default: refs/heads/master
```

Comment:
I can't find a way to get the github default branch name :(

Reply:
Can we just fix "master" for now?

Reply:
I want it to be possible with other candidate too. Like if we need to have a 2.10 patch fix for example. This is not based on master.